app/controllers/v1/content_blocks_controller.rb

```diff
@@ -20,8 +20,15 @@ def index
     end
 
     def store
-      content_path = params[:content_block][:content_path]
-      @content_block = @content_blocks_service.create_or_update(content_path, content_block_params)
+      block_params = params[:content_block]
+      content_path = block_params[:content_path]
+      predecessor_version = block_params[:version] ? block_params[:version].to_i : nil
+      begin
+        @content_block = @content_blocks_service.create_or_update(content_path, content_block_params,
+                                                                  predecessor_version)
+      rescue OutdatedError
```

Comment:
maybe in this case it would make sense to throw an error but then we could use the standard-error with a message, saying that it was outdated, maybe something like this `render text: 'Outdated content', status: 409`.
e.g. http://stackoverflow.com/questions/10048173/why-is-it-bad-style-to-rescue-exception-e-in-ruby